starknet_py/net/models/typed_data.py

```diff
@@ -36,3 +37,8 @@ class TypedDataDict(TypedDict):
     primaryType: str
     domain: DomainDict
     message: Dict[str, Any]
+
+
+class TypeContext(TypedDict):
+    parent: str
+    key: str
```

Comment:
Ideally we would name it `Context`, but there is already a [class with such name](https://github.com/software-mansion/starknet.py/blob/56c60c1bb0ac0b56dd220c597fffef8745845acb/starknet_py/serialization/_context.py#L15).

Reply:
The current name is fine 👍